### PR TITLE
Transactions styles

### PR DESCRIPTION
**Pull request by AngieDutra**

### CODE REVIEW

File: webapp/app/static/jsx/transaction/RevealsTimeline.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default function RevealTimeline({ addr, loadNewData, aliases }) {
     function onSelectWindow(windowOption) {
         setPlotWindow(windowOption);
         getApi(`/plot/transaction?address=${addr}&window=${windowOption}`, response => {
-            // response = exampleResponse;
+            response = exampleResponse;
```
Comment (kkailiwang) [inline webapp/app/static/jsx/transaction/RevealsTimeline.jsx]: can u comment this out please?

**Review by kkailiwang — CHANGES_REQUESTED** [review]
remember to comment out the fake data!

**Comment by kkailiwang** [comment]
image
 this part looks awesome!!

File: webapp/app/static/jsx/components/AgnosticTable.jsx [hunk 2]
```diff
@@ -28,12 +28,12 @@ export default function AgnosticTable({ toIgnore = new Set(), keyValues, aliases
                     }
                     return (
                         <tr className="detail-row" key={idx}>
-                            <td>{k} 
-                            {hovers && hovers[ogK] && <MyTooltip tooltipText={hovers[ogK]} />}
+                            <td>{k}
+                                {hovers && hovers[ogK] && <MyTooltip tooltipText={hovers[ogK]} />}
```
Comment (kkailiwang) [inline webapp/app/static/jsx/components/AgnosticTable.jsx]: Now that there's no spacing between the key and value, it's hard to read that the value corresponds to the key (for example, the heuristic is 'linked transaction' in this case). can you add a ':' at the end of this line, line 32? after the curly bracket? 
image

File: webapp/app/static/jsx/transaction/RevealsTimeline.jsx [hunk 3]
```diff
@@ -88,13 +89,13 @@ export default function RevealTimeline({ addr, loadNewData, aliases }) {
...
-                            //     <div style={{ height: '100%', width: '50px', fill: name === selectedWeek ? '#ee11bb' : '#ff0033' }}></div>
-                            // )}
+                            <Bar name={aliases.torn_mine || 'torn_mine'} dataKey="torn_mine"
+                                // background={({ name }) => (
+                                //     <div style={{ height: '100%', width: '50px', fill: name === selectedWeek ? '#ee11bb' : '#ff0033' }}></div>
+                                // )}
                                 stackId="a" fill="white" />
 
-                            <Legend verticalAlign="bottom" height={5} />
+                            <Legend layout="vertical" verticalAlign="bottom" />
```
Comment (kkailiwang) [inline webapp/app/static/jsx/transaction/RevealsTimeline.jsx]: image
 i don't like how the legend overlaps with the graph - could you make sure it doesn't? I think setting the 'height' attribute helps with that (like `height={5}` .

File: webapp/app/static/jsx/components/AccordionOfResults.jsx [hunk 4]
```diff
@@ -11,10 +11,10 @@ const Spinner = (<div id="spinner" className="justify-content-center">
```
Comment (kkailiwang) [inline webapp/app/static/jsx/components/AccordionOfResults.jsx:11]: image
it looks like the accordionOfResults is mobile-responsive until you expand one of the items. could you fix this?

**Review by kkailiwang — CHANGES_REQUESTED** [review]

File: webapp/app/static/jsx/transaction/RevealsTimeline.jsx (hunk 3 above)
Comment (AngieDutra) [reply webapp/app/static/jsx/transaction/RevealsTimeline.jsx]: Thank you!

**Review by kkailiwang — APPROVED** [review]
beautiful!!!

**Pull request merged**